metadata/requirements-common.txt

```diff
@@ -0,0 +1 @@
+../requirements-common.txt
```

Comment:
nice! Didn't know you could just reference files like this

Reply:
we use it in several places in `docs` subdirectory :)